hyperdrive-cli/commands/stakewise/nodeset/commands.go

```diff
@@ -15,13 +15,20 @@ func RegisterCommands(cmd *cli.Command, name string, aliases []string) {
 			{
 				Name:    "upload-deposit-data",
 				Aliases: []string{"u"},
-				Usage:   "Uploads the combined deposit data for all of your validator keys to NodeSet's Stakewise vault, so they can be assigned new deposits.",
+				Flags: []cli.Flag{
+					&cli.BoolFlag{
+						Name:    "force-upload",
+						Aliases: []string{"f"},
+						Usage:   "Force the upload of the deposit data to the server",
```

Comment:
"Force the upload of all deposit data to the server, bypassing the node wallet balance check"